action.yml

```diff
@@ -27,9 +30,13 @@ runs:
         ref: ${{ github.head_ref }}
     - name: Setup Node.js environment
       uses: actions/setup-node@v2.4.0
-    - run: npx token-transformer ${{ inputs.tokenFile }} ${{ inputs.outputFolder }}/tokens.json --expandTypography=true --expandComposition=true --preserveRawValue=false --resolveReferences=true
+    - run: npm install --prefix ${{ github.action_path }} style-dictionary
```

Comment:
Are we able to use this strategy to just install deps for the repo?

Reply:
I believe so. I just went with style-dictionary since it was the only package that was causing an error to be thrown & I though installing one dependency would be faster than installing them all each time the action is called. I can try installing them all though?

Reply:
Can do.

Reply:
Omitting the dev dependencies reduced installed packages from 375 in 5 seconds down to 57 in 2 seconds. Pretty sweet.